models/classes/CrudResultsService.php

```diff
@@ -45,7 +45,7 @@ public function getRootClass() {
 
     public function get($uri) {
         $returnData = array();
-        $deliveryExecution = new \core_kernel_classes_Resource($uri);
+        $deliveryExecution = \taoDelivery_models_classes_execution_ServiceProxy::singleton()->getDeliveryExecution($uri);
         $delivery = $deliveryExecution->getOnePropertyValue(new \core_kernel_classes_Property(PROPERTY_DELVIERYEXECUTION_DELIVERY));
```

Comment:
Here we should call 
taoDelivery_models_classes_execution_DeliveryExecution::getDelivery()